Userland/Services/AudioServer/Mixer.cpp

```diff
@@ -177,7 +177,8 @@ void Mixer::request_setting_sync()
                 if (auto result = m_config->sync(); result.is_error())
                     dbgln("Failed to write audio mixer config: {}", result.error());
             },
-            this);
+            this)
+                                   .release_value_but_fixme_should_propagate_errors();
```

Comment:
whoa there, what's this about

Reply:
I don't know, but it's how clang-format insists on doing that. Same for the other funky ones.